packages/expansion-hub/src/index.ts

```diff
@@ -0,0 +1,14 @@
+
+export * from "./discovery"
+export * from "./open-rev-hub"
+export * from "./RevHub"
+export * from "./errors/NackError"
+export * from "./errors/UnknownNackError"
+export * from "./errors/i2c-errors"
+export * from "./errors/ParameterOutOfRangeError"
+export * from "./errors/diagnostic-errors"
+export * from "./errors/BatteryTooLowError"
+export * from "./errors/gpio-errors"
+export * from "./errors/motor-errors"
+export * from "./errors/servo-errors"
+export * from "./errors/nack-code-to-error"
```

Comment:
Do we want to export the nack-code-to-error file? This will let users get the exception class for a nack code.

Reply:
Sure